minimesos/src/test/groovy/com/containersol/minimesos/config/ConfigParserTest.groovy

```diff
@@ -17,6 +17,20 @@ public class ConfigParserTest {
     }
 
     @Test
+    public void testLoggingLevel() {
+        String config =
+                """
+        minimesos {
+
+            loggingLevel = "WARNING"
+
+        }
+        """
+
+        assertEquals("WARNING", parser.parse(config).getLoggingLevel())
+    }
+
+    @Test
```

Comment:
Please, add test to ensure possibility to change logging level for specific mesos container.